### PR TITLE
Make the home page look (more) like the designs.

### PR DESCRIPTION
**Pull request by ericroberts**
This is not quite complete, but much more like the designs. The primary issue remaining is the image of the computer screens in the main call to action. I was unable to export it properly from sketch, will need some help from @jetpacksandrollerskates.

Part of #1

Here's what it looks like:

screencapture-localhost-4000-1476408814582

### CODE REVIEW

File: _includes/agplv3.html [hunk 1]
```diff
@@ -0,0 +1,25 @@
+<svg xmlns="http://www.w3.org/2000/svg" viewbox="0 0 864 358" version="1.1">
```
Comment (singpolyma) [inline _includes/agplv3.html]: can this file be named *.svg still and be in includes?

File: _includes/footer.html [hunk 2]
```diff
@@ -1,4 +1,22 @@
...
+	</nav>
+	<section id="newsletter">
+		<form method="post" action="#">
+			<label>
+				<span>Sign up for our newsletter</span>
+				<input type="email" name="email" placeholder="email address" />
+				<button type="submit">Subscribe</button>
+			</label>
+		</form>
+		<p>© {{ site.time | date: "%Y" }} tint. It's a thing by some people that like to make stuff.</p>
```
Comment (singpolyma) [inline _includes/footer.html]: I'm actually opposed to including an old UCC-style copyright notice.  Not sure how you guys feel about it.  I hasn't been legally necessary for my entire life...

File: _includes/logo.html [hunk 3]
```diff
@@ -0,0 +1,7 @@
+<svg viewBox="0 0 64 24" version="1.1" xmlns="http://www.w3.org/2000/svg">
```
Comment (singpolyma) [inline _includes/logo.html]: same thing here -- can the file be named *.svg ?  would be clearer in includes, etc.

File: _includes/agplv3.html (hunk 1 above)
Comment (ericroberts) [reply _includes/agplv3.html]: I thought that I had tried this on another Jekyll site and it didn't work, but I just gave it another try and it worked fine, so I've made that change.

File: _includes/footer.html (hunk 2 above)
Comment (ericroberts) [reply _includes/footer.html]: I agree. Just put it in because it was in the design. It's removed now.

File: _includes/logo.html (hunk 3 above)
Comment (ericroberts) [reply _includes/logo.html]: Done

**Pull request merged**